src/main/java/com/programmers/ticketparis/controller/performance/PerformanceController.java

```diff
@@ -0,0 +1,52 @@
+package com.programmers.ticketparis.controller.performance;
+
+import com.programmers.ticketparis.dto.performance.request.PerformanceCreateRequest;
+import com.programmers.ticketparis.dto.performance.request.PerformanceUpdateRequest;
+import com.programmers.ticketparis.dto.performance.response.PerformanceResponse;
+import com.programmers.ticketparis.service.performance.PerformanceService;
+import lombok.RequiredArgsConstructor;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.List;
+import java.util.Optional;
+
+@RestController
+@RequiredArgsConstructor
+@RequestMapping("/api/performances")
+public class PerformanceController {
+    private final PerformanceService performanceService;
+
+    @PostMapping
+    @ResponseStatus(HttpStatus.CREATED)
+    public void createPerformance(@RequestBody PerformanceCreateRequest createRequest) {
```

Comment:
`p1`

```suggestion
    public void createPerformance(@Valid @RequestBody PerformanceCreateRequest createRequest) {
```

`@Valid`를 사용하지 않는 경우 @RequestBody에 적힌 객체 값을 검증할 수 없습니다! `@Valid`를 추가해주세요!
[Valid 관련 글](https://www.baeldung.com/spring-boot-bean-validation)